src/git-mirror/git-mirror-task.spec.ts

```diff
@@ -285,20 +285,29 @@ describe("GitMirrorTask", () => {
             const pullRef2 = "9eb061c8e73b4c0a08613710daa3f5a93d47061d refs/pull/13/head";
             const mergeRequest1 = "1b60f335a6f3a4bbf7d56a8ae3106f6b3ea77891 refs/merge-requests/10/head";
 
-            const updatedPackedRefsFileContents = "data: # pack-refs with: peeled fully-peeled sorted \\n" +
-            `${branchRef1}\\n${branchRef2}\\n${tagRef1}\\n${tagRef2}\\n`;
```

Comment:
This was silently a bad test before since the actual escape was the backslash token being escaped vs a newline. The tests were passing, but a couple were doing so speciously